README.md

```diff
@@ -62,17 +62,20 @@ Advanced composition
 Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you have a `MyFormInput` component that should also act as both a `Row` and a `Col`.
 
 ```js
-import {getRowClassNames, getColClassNames} from 'react-flexbox-grid'
+import {getRowProps, getColumnProps} from 'react-flexbox-grid'
 
 export default function MyFormInput(props) {
+  const { className: rowClassName } = getRowProps(props);
+  const { className: colClassName, ...myProps } = getColumnProps(props);
```

Comment:
yes, good point regarding avoiding renaming a de-structured property.

Really the advantage of exporting the prop generating functions is so you can wrap third party components that only accept a `className`.  I've updated the example to show that.

Another small advantage is that the new exports allow you to split the components that are for row/columns out from ones that your custom component needs.  If you just pass the entire props onto `Row` or `Column` then you end up with React warnings about unknown props.